OpenRA.Mods.Common/Traits/Conditions/SpreadsCondition.cs

```diff
@@ -0,0 +1,72 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Linq;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Any actor with this trait enabled has a chance to affect others with this trait.")]
+	public class SpreadsConditionInfo : ConditionalTraitInfo
+	{
+		[Desc("The chance this actor is going to affect an adjacent actor.")]
+		public readonly int Probability = 10;
```

Comment:
This might be a bit large. We could also increase the delay. On the other hand I wouldn't really mind leaving it as-is, it just felt like the fire spreads quite quickly.